decentraland/unity-renderer
# Feat: allow to participate event or remove participation from the explorer

## What does this PR change?

Allow users to "want to go" to events from the explorer without opening the website

...

## How to test the changes?


1. Launch the explorer https://play.decentraland.org/?explorer-branch=feat/event-want-to-go
2. Go to event page
3. In a future event click "Want to go"
4. Check it changes to going
5. Check that in the events website https://events.decentraland.org/me/ the event is added to the list
6. Do the same with the other way around, removing the going to the event

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
